components/Navigation/index.js

```diff
@@ -28,29 +34,29 @@ export default Navigation;
 //styled-components
 
 const StyledFooter = styled.footer`
-  font-size: 0, 5em;
-  position: fixed;
-  bottom: 0;
-  color: black;
-  width: 100%;
-  height: 70px;
-  border: 1px solid black;
+  bottom: 0px;
+  background-color: #6495ed;
+  postion: relative; 
+  color: darkblue;
+  line-height: 1rem;
+  padding: 0px;
+  margin: 0px;
+  border: 5px solid lightblue;
+  border-radius: 1em;
```

Comment:
This gets me confused a little bit :D why is it that this whole file has no syntax highlighting?

Reply:
What do you mean excactly with "syntax highlighting" and how can i improve it?

Reply:
Syntax highlighting is the ability of a computer program to display certain words and character combinations in a text in different colors, fonts, and styles depending on their meaning. 

Github is not doing that for this file, this is confusing